http_request_translator/plugin_manager.py

```diff
@@ -1,27 +1,22 @@
 from __future__ import print_function
 
-from .script import BashScript, PHPScript, PythonScript, RubyScript
+from .base import AbstractScript
 
 
-def hardcoded_script_import(script_name):
-    """Manually find the class of the script.
-
-    :param str script_name: Name of the script from which to import the function.
+def get_script_class(script_name):
+    """Returns the class of the script.
 
+    :param str script_name: language name of the script class which we want to import
     :raises ValueError: When the script is not supported.
-
     :return: The class of the `script_name` script.
-    :rtype: :class:`AbstractScript`
+    :rtype: :class: `AbstractScript`
```

Comment:
There shouldn't be a space between  `:class:` and the class.Otherwise Sphinx, used to generate the documentation, will fail to find the reference.